src/options.h

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright (c) 2017-2020, Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+//////////////////////////////////////////////////////////////////////////////
+#pragma once
+
+#include "enums.h"
+#include <cstdlib>
+#include <functional>
+#include <map>
+#include <tuple>
```

Comment:
include <string> here to fix build problem on macOS.

Reply:
Fixed